hi-ml-multimodal/notebooks/phrase_grounding.ipynb

```diff
@@ -101,8 +103,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "text_inference = get_cxr_bert_inference()\n",
-    "image_inference = get_biovil_resnet_inference()"
+    "text_inference = get_bert_inference(\"biovil_t_bert\")\n",
```

Comment:
Should we use the enums `ImageModelType` and `BertEncoderType` here?

Reply:
Done in 714552e7.